### PR TITLE
Add Filter icon

### PR DESCRIPTION
**Pull request by rosslo**
Add a Filter icon

### CODE REVIEW

**Comment by franciscrispin** [comment]
WIP. Waiting on gun for the resized icon path.

**Comment by rosslo** [comment]
Updated. @franciscrispin please help review, thanks!

File: src/Utils/IconLibrary.tsx [hunk 1]
```diff
@@ -52,6 +52,7 @@ export const ICONS = {
...
   FILE_SOLID: <path d="M54.072 34.976h27.765L54.072 7.452v27.524zM19.096 0H59.24l30.048 30.048v60.096c0 2.725-1.001 5.048-3.004 6.971C84.28 99.038 81.917 100 79.192 100H19.096c-2.724 0-5.088-.962-7.091-2.885C10.002 95.192 9 92.87 9 90.145l.24-80.29c0-2.724.962-5.047 2.885-6.97C14.048.962 16.372 0 19.096 0z" />,
+  FILTER: <path d="M75,70 C78.3137085,70 81,72.6862915 81,76 L81,79.357 L100,79.3571429 L100,90.5 L81,90.499 L81,94 C81,97.3137085 78.3137085,100 75,100 L68,100 C64.6862915,100 62,97.3137085 62,94 L62,90.499 L0,90.5 L0,79.3571429 L62,79.357 L62,76 C62,72.6862915 64.6862915,70 68,70 L75,70 Z M32,35 C35.3137085,35 38,37.6862915 38,41 L38,44.357 L100,44.3571429 L100,55.5 L38,55.499 L38,59 C38,62.3137085 35.3137085,65 32,65 L25,65 C21.6862915,65 19,62.3137085 19,59 L19,55.499 L0,55.5 L0,44.3571429 L19,44.357 L19,41 C19,37.6862915 21.6862915,35 25,35 L32,35 Z M75,0 C78.3137085,0 81,2.6862915 81,6 L81,9.357 L100,9.35714286 L100,20.5 L81,20.5 L81,24 C81,27.3137085 78.3137085,30 75,30 L68,30 C64.6862915,30 62,27.3137085 62,24 L62,20.5 L0,20.5 L0,9.35714286 L62,9.357 L62,6 C62,2.6862915 64.6862915,0 68,0 L75,0 Z" id="path-1" />,
```
Comment (franciscrispin) [inline src/Utils/IconLibrary.tsx]: Oh @rosslo you should be able to remove the `id="path-1"` attribtue

Comment (rosslo) [reply src/Utils/IconLibrary.tsx]: Updated!

**Pull request merged**